### PR TITLE
#163518685 Mock SendGrid Mail

### PR DESCRIPTION
**Pull request by abayo-luc**
#### What does this PR do?
Disable sending reset password email from test
#### Description of Task to be completed?
```POST /api/users/reset_password```
#### How should this be manually tested?
-Clone the repo and check out `bg-moch-sendgrid-mail-163518685` branch
- Go under ```helpers``` directories, and change dummyUser email to your own working email.
- Run ```yarn test```, and check your email, you will see nothing. 
#### Any background context you want to provide?
before running anything, ensure packages are updated ```yarn``` 
#### What are the relevant pivotal tracker stories?
163518685

### CODE REVIEW

**Review by niomwungeri-fabrice — APPROVED** [review]
Good Job mate!

File: tests/users/passwordReset.test.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
 describe("/API end point /users/rese_password", () => {
   before(async () => {
     await chai
       .request(app)
       .post("/api/v1/users")
       .send({
-        ...dummyUser2
+        ...dummyUser
       });
+    nock("https://api.sendgrid.com")
```
Comment (espoirMur) [inline tests/users/passwordReset.test.js:21]: Nice job

**Review by espoirMur — APPROVED** [review]

File: helpers/data.js [hunk 2]
```diff
@@ -10,3 +10,49 @@ export const data = {
     username: "jean786786"
   }
 };
+
```
Comment (espoirMur) [inline helpers/data.js:13]: Can you please move this file into test/helpers/ as the sendGridResponse object is a test data?

**Review by espoirMur — CHANGES_REQUESTED** [review]

**Review by espoirMur — APPROVED** [review]

**Review by niomwungeri-fabrice — APPROVED** [review]

File: models/Article.js [hunk 3]
```diff
@@ -11,9 +11,26 @@ export default (sequelize, DataTypes) => {
...
+      readTime: {
+        type: DataTypes.INTEGER,
+        allowNull: true,
+        defaultValue: 0
+      }
     },
-    { tableName: "articles" }
+    {
+      tableName: "articles"
+      // hooks: {
```
Comment (puritybirir) [inline models/Article.js]: Why are these comments going to staging?

**Review by puritybirir — CHANGES_REQUESTED** [review]

**Comment by puritybirir** [comment]
Name of the Branch should also be bg-mock-sendgrid and not bg-moch-sendgrid

File: models/Article.js (hunk 3 above)
Comment (abayo-luc) [reply models/Article.js]: @puritybirir that one was from develop, but now I removed it.

**Comment by abayo-luc** [comment]
Duplicate of #31

**Pull request closed without merging**